### PR TITLE
Issue#60 pds3 image object from numpy array

### PR DESCRIPTION
**Pull request by bvnayak**

### CODE REVIEW

**Comment by godber** [comment]
Good progress @bvnayak!  My initial reactions are

1. Perhaps the interface for array data should be `PDS3Image(data)` instead of  `PDS3Image.open(data)`.  Is there any benefit to using open?
2. I think your test would seem more direct and be more informative to users if you manually created the `numpy` array rather than reading it from a file.
3. More tests could be implemented to test other parts of the code by creating a variety of different arrays.

**Comment by wtolson** [comment]
@godber That's similar to what I was thinking: https://github.com/wtolson/planetaryimage/blob/plugin-refactor/planetaryimage/image.py#L26

**Comment by godber** [comment]
@wtolson ... Bhavin and I have both played with that branch at this point and may fiddle with it further.  We're addressing some internal needs right now and then will regroup and look at your plugin based approach.

**Comment by bvnayak** [comment]
More tests on different `dtypes` are coming soon !

File: planetaryimage/pds3image.py [hunk 1]
```diff
@@ -138,8 +153,9 @@ def _save(self, file_to_write, overwrite):
 
         if self._sample_bytes != self.data.itemsize:
             self.label['IMAGE']['SAMPLE_BITS'] = self.data.itemsize * 8
-            sample_type_to_save = self.DTYPES[self._sample_type[0] + self.dtype.kind]
-            self.label['IMAGE']['SAMPLE_TYPE'] = sample_type_to_save
+
+        sample_type_to_save = self.DTYPES[self._sample_type[0] + self.dtype.kind]
```
Comment (bvnayak) [inline planetaryimage/pds3image.py]: While writing a test, I realized that these 2 changed lines does not belong to the if loop. (When you create a PDS3Image object from bumpy array with `sample_type` other than `MSB_INTEGER`, you'll get an error if these lines are inside the if loop.)

**Pull request merged**